pyproject.toml

```diff
@@ -45,31 +45,35 @@ testpaths = "cf_units"
 write_to = "cf_units/_version.py"
 local_scheme = "dirty-tag"
 
-[tool.black]
+[tool.ruff]
 line-length = 79
-target-version = ["py39", "py310", "py311"]
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs         # exclude a few common directories in the
-    | \.git          # root of the project
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-  )/
-  | _version.py
-)
-'''
+target-version = "py38"
```

Comment:
Since Python 3.8 will be unsupported soon, I think this should be updated to a later version. Is there a reason for decreasing the version here or is this just a symptom of being an old PR?

Reply:
Please note:

https://github.com/SciTools/cf-units/blob/539dbccd5597b73c1785bbd31d51e4c304e7ffc1/cf_units/tests/test_coding_standards.py#L121-L122

Reply:
> Is there a reason for decreasing the version here or is this just a symptom of being an old PR?

Almost certainly just old PR.